allennlp/predictors/multitask.py

```diff
@@ -0,0 +1,102 @@
+from overrides import overrides
+from typing import Type, List, Dict
+
+from allennlp.common import JsonDict
+from allennlp.data import DatasetReader, Instance
+from allennlp.models import Model
+from allennlp.predictors.predictor import Predictor
+
+
+@Predictor.register("multitask")
+class MultiTaskPredictor(Predictor):
+    """
+    Predictor for multitask models.
+
+    Registered as a `Predictor` with name "multitask".
+    """
```

Comment:
I think we should also state here that it only works with `MultiTaskDatasetReader` and `MultiTaskModel`